### DISCUSSION TITLE
Sliver doesn't generate payload

### DISCUSSION THREAD

**Discussion by elchinsir** [BishopFox/sliver#1434] [Q&A]
Hey guys, I need the help with this error. Somehow the command doesn't generate the exe file. Any suggestion?
This path is empty: open /root/.sliver/slivers/windows/amd64/COSTLY_CHANGE/bin/
Sliver > generate -m 192.xxx.xxx.xx5 -e

[*] Generating new windows/amd64 implant binary
[*] Symbol obfuscation is enabled.
[*] This process can take awhile, and consumes significant amounts of CPU/Memory
[!] rpc error: code = Unknown desc = open /root/.sliver/slivers/windows/amd64/COSTLY_CHANGE/bin/COSTLY_CHANGE.exe: no such file or directory
![sliver](https://github.com/BishopFox/sliver/assets/96763809/79b2cd5b-60f6-499b-b3ec-2589af73e444)

**Comment by moloch--** [comment]
Try `sudo apt install build-essential git mingw-w64 binutils-mingw-w64 g++-mingw-w64` this error usually means you're missing `git` or MinGW. Check the logs for more details if this doesn't work (should be in `~/.sliver/logs/sliver.log`

**Reply by moloch--** [reply]
(we probably need to add a more friendly error message)

**Reply by rkervella** [reply]
`mingw` shouldn't be required for non-DLL builds though, it might be something else. How did you install Sliver @elchinsir ?

**Reply by elchinsir** [reply]
Step by step:

1- sudo apt-get install mingw-w64 binutils-mingw-w64 g++-mingw-w64

2- wget https://github.com/BishopFox/sliver/releases/download/v1.4.14/sliver-server_linux.zip

3- gzip sliver-server_linux.zip 
                                                                                         
4- chmod +x sliver-server
                                                                                         
5- sudo ./sliver-server
Unpacking assets ...

All hackers gain exalted
[*] Server v1.4.14 - 6c02971b54831884d30407b632a379947dd289ad - Dirty
[*] Welcome to the sliver shell, please type 'help' for options

[*] Check for updates with the 'update' command

All good to here.

When I use generate command, it gives error:

sliver > generate -m 192.168.199.135 -e

[*] Generating new windows/amd64 implant binary
[*] Symbol obfuscation is enabled.
[*] This process can take awhile, and consumes significant amounts of CPU/Memory
[!] rpc error: code = Unknown desc = open /root/.sliver/slivers/windows/amd64/CIRCULAR_TV/bin/CIRCULAR_TV.exe: no such file or directory

basically it creates the path but doesnt generate the payload. 

![seliver2](https://github.com/BishopFox/sliver/assets/96763809/5d39936b-404f-468a-80ad-e077566dfa55)

# cd /root/.sliver/slivers/windows/amd64/CIRCULAR_TV/bin/               
                                                                                                                                                                                                                   
# ls
EMPTY

**Reply by rkervella** [reply]
Any reason you're using a super old version?

**Reply by elchinsir** [reply]
Do you minD providing me thE link to the most recent version for Linux? (KALI)

**Reply by rkervella** [reply]
The top of this page should always have the latest version: https://github.com/bishopfox/sliver/releases
